test/hotspot/jtreg/compiler/loopopts/superword/TestEquivalentInvariants.java

```diff
@@ -0,0 +1,951 @@
+/*
+ * Copyright (c) 2024, 2025, Oracle and/or its affiliates. All rights reserved.
```

Comment:
This is new file. Why two years?

Reply:
Nice catch. I must have just blindly extended it when I updated the copyright year.